airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/spec.py

```diff
@@ -31,6 +31,8 @@
 DATE_TIME_PATTERN = "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$"
 EMPTY_PATTERN = "^$"
 
+TimeIncrement = Union[PositiveInt, Literal["monthly"]]
```

Comment:
This can be tricky to have this complex data type. Probably it would be easier to implement a new parameter to switch the range type from days to monthly.
@maxi297 what are your thoughts?

Reply:
Any updates here? So @marcosmarxm should I change the parameter?